.circleci/config.yml

```diff
@@ -66,12 +66,12 @@ jobs:
           keys:
             # First, try finding a cache entry with the same set of libraries
             # that also comes from the same branch (so as to make the best use of Gradle compilation cache).
-            - gradle-deps-v5-{{ checksum "gradle/libs.versions.toml" }}-{{ .Branch }}
+            - gradle-deps-v6-{{ checksum "gradle/libs.versions.toml" }}-{{ .Branch }}
```

Comment:
To start with clean slate, in order to check if all the Maven repo config is okay

Reply:
while we are here... in the most common scenario we need to update three lines here. can we have smth like `gradle-deps-v{{ version-number }}` instead? so only a single variable changes? 🤔

Reply:
Yeah, circleci supports `<< ... >>` -style parameters... but it then looks even more confusing then now, I'd rather stick to the current looks 😅